test/controllers/AreAnySealsBrokenControllerSpec.scala

```diff
@@ -40,7 +45,8 @@ class AreAnySealsBrokenControllerSpec extends SpecBase with AppWithDefaultMockFi
 
     "must return OK and the correct view for a GET" in {
       checkArrivalStatus()
-
+      when(mockUnloadingPermissionMessageService.getMessageHead(any())(any(), any()))
+        .thenReturn(Future.successful(Some(MessageMetaData(LocalDateTime.now(), UnloadingPermission, ""))))
```

Comment:
Could this not be done via the AppWithDefaultMockFixtures. May save you having to add this to each controller